src/detections/table/body/index.jsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react'
 
-import { detections } from 'detections/table/body/data'
 import { Row } from 'detections/table/body/row'
+import { useSelector } from 'react-redux'
+import { get } from 'lodash'
 
-export const TableContent = () => detections.map((item, index) => <Row item={item} key={index} />)
+export const TableContent = () => {
+  const items = useSelector(({ detections }) => get(detections, 'get.data.items', []))
```

Comment:
Let's also extract this value to a separate hook.

Reply:
Done